src/lib.rs

```diff
@@ -37,6 +37,15 @@ use derive_builder::Builder;
 use font8x8::UnicodeFonts;
 use ratatui::{prelude::*, text::StyledGrapheme, widgets::Widget};
 
+#[derive(Debug, Clone, Copy, Eq, PartialEq, Hash, Default)]
+pub enum BigTextSize {
+    #[default]
+    Full,
+    HalfHeight,
+    HalfWidth,
+    Half,
```

Comment:
I am not sure about the naming.
The one currently named HalfHeight really seems to look more natural, but on the other hand what would the one currently named HalfWidth be named then without being too long or complicated.

If we come up with a good set of new names, I'd like to use those.

Reply:
I now renamed Half to Quadrant to more precisely match what portion of a cell as pixel from the 8x8 font uses.  
I would say, the naming Full, HalfWidth, HalfHeight and Quadrant is describing the "pixel size" quite well, what do you think?

This would also allow to name a possible future pixelsize with 6 pixels per cell (using `'🬗'` and others) as something like `PixelSize::Sextant`.